src/components/List.js

```diff
@@ -1,12 +1,54 @@
 import React from 'react';
 import useFirestore from '../hooks/useFirestore';
+import { db } from '../lib/firebase';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+
+  const checkPurchasedDate = (purchasedDate) => {
+    if (purchasedDate === null) {
+      return false;
+    }
+    const today = new Date();
+    let daySincePurchased = purchasedDate.toDate();
```

Comment:
Really interesting approach here :-) I really like the idea of adding a day to the purchase day and then comparing that to today.

I think that the variable naming is a little confusing though. We're calling this variable `daysSincePurchased`, so I would expect it to be an integer indicating how many days have passed since it was purchased. However, it's actually a firebase timestamp of purchased date that is converted to a date object. Then, when we mutate it, it becomes that date plus one day.
I wonder if the team has any ideas for what would be a some good names here?

PS - one of my favorite programming quotes is: 

"There are only two hard things in Computer Science: cache invalidation and naming things."

-- Phil Karlton

Reply:
For ideas on variable names, maybe something regarding a timestamp, like timestampOfPurchasedDate or timestampOfCurrentDate (those are a little verbose, but something in that realm or ballpark maybe) ?

Reply:
I agree, the naming conventions are a bit confusing, I like your suggestion Brittany